web/packages/quick-editor/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export type {
+	Scope,
+	ProfileUpdatedType,
+	OnProfileUpdated,
+	OnOpened,
+	QuickEditorCoreOptions
+} from './quick-editor-core';
+
+export type { QuickEditorOptions } from './quick-editor';
+
+export { default } from './quick-editor';
```

Comment:
My idea here was to make the `GravatarQuickEditor` available as the default module, allowing developers to import it like `import GravatarQuickEditor from '@gravatar-com/quick-editor';
But that's not mandatory, I just thought it would make sense to have a default module.

Reply:
I have changed it as you suggested above.